JavaSource/org/unitime/timetable/util/InstructionalOfferingRollForward.java

```diff
@@ -472,6 +474,7 @@ private InstructionalOffering createToInstructionalOfferingFromFromInstructional
 			toCourseOffering.setReservation(fromCourseOffering.getReservation());
 			toCourseOffering.setConsentType(fromCourseOffering.getConsentType());
 			toCourseOffering.setCourseType(fromCourseOffering.getCourseType());
+			toCourseOffering.setFundingDept(fromCourseOffering.getFundingDept());//TODO
```

Comment:
This does not work. You need to look up the matching funding department in the toSession